Source/App/EbAppInputy4m.c

```diff
@@ -320,15 +333,15 @@ EB_BOOL check_if_y4m(EbConfig_t *cfg) {
     }
 
     buffer[YUV4MPEG2_IND_SIZE] = 0;
-    if (validateAlphanumeric((char*)buffer) && EB_STRCMP(buffer, "YUV4MPEG2") == 0) {
+    if (validateAlphanumeric((char*)buffer) && EB_STRCMP((char*)buffer, "YUV4MPEG2") == 0) {
```

Comment:
both y4m_buf, buffer are unsigned char type. I don't think you need conversion.

Reply:
validateAlphanumeric(char* buffer)==>validateAlphanumeric(unsigned char* buffer)